lib/app-capitano.coffee

```diff
@@ -154,5 +154,17 @@ exports.run = (argv) ->
 		else
 			capitanoExecuteAsync(cli)
 
-	Promise.all([events.trackCommand(cli), runCommand()])
+	trackCommand = ->
+		getMatchCommandAsync = Promise.promisify(capitano.state.getMatchCommand)
+		getMatchCommandAsync(cli.command)
+		.then (command) ->
+			# cmdSignature is literally a string like, for example:
+			#     "push <applicationOrDevice>"
+			# ("applicationOrDevice" is NOT replaced with its actual value)
+			# In case of failures like an inexistent or invalid command,
+			# command.signature.toString() returns '*'
+			cmdSignature = command.signature.toString()
+			events.trackCommand(cmdSignature)
```

Comment:
This code block (`trackCommand` function) was moved here from `events.ts` because it is specific to Capitano (`event.ts` was modified for use with oclif as well).